app/models/student.rb

```diff
@@ -1,4 +1,10 @@
 class Student < ApplicationRecord
+  VALID_EMAIL_REGEX  = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
+  VALID_PHONE_REGEX  = /\A\([0-9]{3}\) [0-9]{3}-[0-9]{4}\z/
+  VALID_ZIP_REGEX    = /\A[0-9]{5}\z/
+  VALID_SMARTT_REGEX = /\A[0-9]{4}-[0-9]{6}\z/
+  LAST_NAME_ID_REGEX = /\A[0-9]{4,5}\z/
```

Comment:
I know you didn't make this up, but I think it's weird that some of these start with `VALID`, as if we ever wanted to use `INVALID` regexes.